src/index.css

```diff
@@ -91,6 +91,32 @@
 .pointer {
   cursor: pointer;
 }
+
+.signup-box {
```

Comment:
you could have used the utility class.

Reply:
sure i will do this after assignment review